### ISSUE TITLE
Where to get mozilla-rust-sdk

### ISSUE THREAD

**Issue by Mic92** [mozilla-services/syncstorage-rs#355]
The [Cargo.toml](https://github.com/mozilla-services/syncstorage-rs/blob/f62f276c49b306fdf929b6f1a778fd9cc7645a52/Cargo.toml#L34) points to it, but it is unclear, where one can download it.

**Comment by jrconlin** [comment]
mozilla-rust-sdk is a rust based google cloud platform library we worked with google to create. We are working with them to get this published under their repos, and hope that they do so soon. We may vendor an early release of the library under our code base, but would rather that the official library be used. Sadly, we have no real insight into their publishing timetable.

**Comment by jheiselman** [comment]
Just so we're clear, that means that a regular user cannot build this package at this time. Is that correct? I just want to make sure that I'm not wasting time trying to get this working before it's ready for users to test out.

**Comment by Mic92** [comment]
No, you cannot build this package without this dependency.

**Comment by jrconlin** [comment]
We're currently debating a few options about what to do. Honestly, the release for that library is taking longer than we expected and we understand the frustration. We've [opened a ticket](https://github.com/mozilla-services/services-engineering/issues/23) in a more general engineering area, but the two options we're considering are:

1) Self Host gets Mysql support only.
This probably maps pretty well with what people prefer, but since we're a small team, we'll need to do a bit of work to make sure that the mysql side matches up with the changes on the spanner side (schema updates, batch behaviors, etc.) Spanner is not mysql, in spite of the press, so there are subtle changes we've had to make.

2) Vendor the spanner library with lots of warning flags
We're not going to be in a position to support the library, and we already realize that the version we're using is not the final form. We could vendor it so folks can compile at least, but it's probably not going to help a lot of self host folk anwyay.

We're leaning toward option #1, but we've got a deadline crunch we're trying to meet as well. 

Still, if y'all have opinions or preferences, let us know.
